pom.xml

```diff
@@ -78,9 +78,9 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <maven.compiler.source>1.8</maven.compiler.source>
-        <maven.compiler.target>1.8</maven.compiler.target>
-        <netbeans.hint.jdkPlatform>JDK_1.8</netbeans.hint.jdkPlatform>
+        <maven.compiler.release>11</maven.compiler.release>
```

Comment:
`maven.compiler.release` seems to be the new recommended option instead of `maven.compiler.source` and `maven.compiler.target`:

https://www.baeldung.com/java-compiler-release-option#conclusion